ui/apps/platform/src/Containers/VulnMgmt/Reports/VulnMgmtReportForm.tsx

```diff
@@ -193,12 +193,15 @@ function VulnMgmtReportForm({
     }
 
     function onScheduledRepeatChange(_id, selection) {
-        // zero out the days selected list if changing interval type
-        if (selection !== values.schedule.intervalType) {
-            void setFieldValue('schedule.interval.days', []);
```

Comment:
I'm guessing `schedule.interval` is a vestigial property since it doesn't exist on the `Schedule` type or the backend proto, and this line hasn't been edited in ~15 months.